BUILD

```diff
@@ -62,12 +63,30 @@ objc_library(
     visibility = ["//visibility:private"],
 )
 
-ios_unit_test(
+ios_test_runner(
+    name = "IPAD_PRO_12_9_IN_9_3",
+    device_type = "iPad Pro (12.9-inch)",
+    os_version = "9.3",
+    visibility = ["//visibility:public"],
+)
+
+ios_test_runner(
+    name = "IPHONE_7_PLUS_IN_10_3",
+    device_type = "iPhone 7 Plus",
+    os_version = "10.3",
+    visibility = ["//visibility:public"],
+)
+
+ios_unit_test_suite(
     name = "UnitTests",
     deps = [
       ":UnitTestsLib",
     ],
-    minimum_os_version = "8.0",
+    minimum_os_version = "9.0",
     timeout = "short",
+    runners = [
+        ":IPAD_PRO_12_9_IN_9_3",
+        ":IPHONE_7_PLUS_IN_10_3",
```

Comment:
Tip: Since these runtimes are being dropped by Xcode 11, do we want to instead migrate to iOS 11/12/13?  Perhaps at least add a newer iOS version if we need to keep iOS 9?

Reply:
Done.